### PR TITLE
update community guides

### PR DESCRIPTION
**Pull request by parkervcp**
changes the community guides to specify 0.6.X versus wings.

Add Ubuntu 20.04

### CODE REVIEW

**Comment by Sir3lit** [comment]
You against using 
`apt -y install php7.4 php7.4-{cli,gd,mysql,pdo,mbstring,tokenizer,bcmath,xml,fpm,curl,zip}` 
over
`apt install -y php7.4 php7.4-cli php7.4-gd php7.4-mysql php7.4-pdo php7.4-mbstring php7.4-tokenizer php7.4-bcmath php7.4-xml php7.4-fpm php7.4-curl php7.4-zip`

**Comment by parkervcp** [comment]
I mean it works either way. The way I wrote it is in line with everything else I have.

**Comment by Sir3lit** [comment]
Just one less command people would have to run, if we used ` systemctl enable --now` instead of `systemctl enable` then `systemctl start`

**Comment by parkervcp** [comment]
If you are copy/pasting does it really matter all that much?

File: community/installation-guides/panel/ubuntu2004.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```
+
+### PHP 7.4
+
+```bash
+## Get apt updates
+apt update -y
+
+## Install PHP 7.4
+apt install -y php7.4 php7.4-cli php7.4-gd php7.4-mysql php7.4-pdo php7.4-mbstring php7.4-tokenizer php7.4-bcmath php7.4-xml php7.4-fpm php7.4-curl php7.4-zip
```
Comment (Sir3lit) [inline community/installation-guides/panel/ubuntu2004.md:33]: `php7.4-pdo` + `php7.4-tokenizer` can be removed and replaced with `php7.4-common` as they're apart of the common package now.

File: community/installation-guides/panel/ubuntu2004.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+apt install -y nginx
+```
+
+### Redis
+
+```bash
+apt install -y redis-server
+
+systemctl start redis-server
+systemctl enable redis-server
```
Comment (Sir3lit) [inline community/installation-guides/panel/ubuntu2004.md:48]: Could remove these as they're at the end of making the pteroq.service file in the main docs, that you direct them to go once done with the redis setup at the bottom

Comment (parkervcp) [reply community/installation-guides/panel/ubuntu2004.md:48]: Commands are different between between Cent and Ubuntu I call it out now out of habit. Still good to have around.

Comment (matthewpi) [reply community/installation-guides/panel/ubuntu2004.md:48]: You could also change it to `systemctl enable --now redis-server` to run both `start` and `enable` in one command.

Comment (Sir3lit) [reply community/installation-guides/panel/ubuntu2004.md:48]: As this is specifically for Ubuntu 20.04... I see nothing wrong with using `systemctl enable --now <service>` for the services

Comment (parkervcp) [reply community/installation-guides/panel/ubuntu2004.md:48]: As previously stated.

image

**Comment by Blamo27** [comment]
@Sir3lit You can send mails with the new version of php ?

I got this error :
`production.ERROR: ErrorException: Trying to access array offset on value of type int in /var/www/pterodactyl/vendor/symfony/console/Input/ArrayInput.php:135`

**Comment by Blamo27** [comment]
I just composer update -v
after changing 
`
"require": {
        "php": ">=7.4",
         ...
 }
`
And the mail send / mail save works fine now.

**Pull request merged**